libraries/common/constants/Portals.js

```diff
@@ -20,6 +20,11 @@ const HOME = 'home';
 const TERMS = 'terms';
 const PRIVACY = 'privacy';
 const IMPRINT = 'imprint';
+const MY_ORDERS = 'my-orders';
```

Comment:
Could you please prefix those with `USER_`, eg. `USER_ORDERS`, `USER_DETAILS` , `USER_ADDRESS_BOOK`, `USER_CHANGE_PASSWORD`